curator-recipes/src/main/java/org/apache/curator/framework/recipes/leader/LeaderLatch.java

```diff
@@ -190,6 +190,12 @@ public void close() throws IOException
         close(closeMode);
     }
 
+    // for testing
```

Comment:
Use `@VisibleForTesting`

Reply:
OK. cc @XComp as multiple reviewers ask for it, I'm making the change now :)